frontend/src/metabase/lib/urls/questions.ts

```diff
@@ -105,9 +107,7 @@ export function newQuestion({
     creationType,
     query: objectId ? { objectId } : undefined,
   });
-
-  const entity = question.isDataset() ? "model" : "question";
-  const type = question.type() ?? entity;
+  const type = question.type();
```

Comment:
The fallback is no longer needed thanks to 98b8579223391532c87276f89acbf271012c8c5f